index.html

```diff
@@ -406,14 +406,14 @@ <h2 class="section-title">Our Philosophy</h2>
                 <div class="philosophy-item">
                     <h3 class="philosophy-title">Raison d'être</h3>
                     <p class="philosophy-text">We believe FreeBSD is perfect for a NAS.</p>
-                    <p class="philosophy-text">We do not believe that Hyperconvergence is the solution for everyone.</p>
-                    <p class="philosophy-text">We harbor no ill-will towards iXsystems. We appreciate the work that they have done in the community for years and wish them the best of luck in their endeavors with TrueNAS SCALE.</p>
+                    <p class="philosophy-text">We do not believe that hyperconvergence is the solution for everyone.</p>
```

Comment:
```suggestion
                    <p class="philosophy-text">We do not believe that hyper-convergence is the solution for everyone.</p>
```

_hyperconvergence_ is detectably misspelt. Hyphenate, consistent with: 

- https://github.com/zvaultio/website/pull/2